middleware/slow_weblogic_start.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Slow WebLogic Starts
+layout: en
+permalink: /posts/middleware/slow_weblogic_starts/
+---
+
+If Weblogic is booting slowly on Linux, it might be a low entropy situation. If entropy is less than 1000, 
+
+```bash
+cat /proc/sys/kernel/random/entropy_avail
+```
+
+you should install a tool to help generate more entropy.
+
+## haveged
+
+[`haveged`](https://issihosts.com/haveged/) is one tool to help generate entroy on Linux.
+
+```bash
+sudo rpm -Uvh https://download.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
```

Comment:
Please don't encourage the continued use of RHEL 6 :)

Reply:
It's what I've got for now, but fair point, I'll add a "Linux 7" section too.